tests/e2e/specs/in-game/spectating/replays.spec.js

```diff
@@ -26,17 +26,28 @@ describe('Rewatching finished games', () => {
 
     beforeEach(createAndFinishCasualMatch);
 
-    it('Rewatches someone else\'s casual match', function () {  
+    it('Rewatches another players casual match as pNum0', function () {
```

Comment:
Nitpick: we call them p0 and p1 for the players 
```suggestion
    it('Rewatches another players casual match as p0', function () {
```